TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Drivetrain.java

```diff
@@ -0,0 +1,36 @@
+package org.firstinspires.ftc.teamcode;
+
+import static com.qualcomm.robotcore.hardware.DcMotorSimple.Direction.FORWARD;
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.gamepad1;
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.hardwareMap;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+
+
+public class Drivetrain {
+    DcMotor moaterFL;
+    DcMotor moaterFR;
+    DcMotor moaterBL;
+    DcMotor moaterBR;
+
+    public void init(){
+        moaterFL = hardwareMap.dcMotor.get("motorFL");
+        moaterFR = hardwareMap.dcMotor.get("motorFR");
+        moaterBL = hardwareMap.dcMotor.get("motorBL");
+        moaterBR = hardwareMap.dcMotor.get("motorBR");
+        moaterFL.setDirection(FORWARD);
```

Comment:
1 side of the drivetrain needs to be flipped (It doesint matter what side needs to be flipped)